language/Dutch/strings.po

```diff
@@ -2921,15 +2921,16 @@ msgstr ""
 
 #: plugins/cheatfind/init.lua:504
 msgid "Pause Mode"
-msgstr ""
+msgstr "Pauzeermodus"
 
 #: plugins/cheatfind/init.lua:511
 msgid "Manually pause & unpause the game when needed with the pause hotkey"
 msgstr ""
+"Handmatig het spel wanneer nodig pauzeren & hervatten met de pauze sneltoets"
```

Comment:
& = en?
(I'm unsure if usage of "&" is correct in English here either)

Reply:
Indeed, I translated the original string. But I'll fix it now